bug: saucenao panic

bot:
```bash
[DEBUG] [ws] 向服务器发送请求: &{send_group_msg map[group_id:820309545 message:少女折寿中...] 103}
[DEBUG] [ws] 接收到API调用返回: {"status":"ok","retcode":0,"data":{"message_id":573842431},"echo":103} 
[INFO] [api] 发送群消息(820309545): 少女折寿中... (id=573842431) 
[ERROR] [bot] execute handler err: interface conversion: http.RoundTripper is nil, not *http.Transport
goroutine 29130 [running]:
runtime/debug.Stack()
        C:/Program Files/Go/src/runtime/debug/stack.go:26 +0x5e
github.com/wdvxdr1123/ZeroBot.match.func2.1.1()
        C:/Users/Komei_DiSanXian/go/pkg/mod/github.com/wdvxdr1123/!zero!bot@v1.7.5-0.20240501144516-eb574bbdad32/bot.go:253 +0x39
panic({0x14a2dc0?, 0xc0027d6450?})
        C:/Program Files/Go/src/runtime/panic.go:785 +0x132
github.com/FloatTech/floatbox/web.NewTLS12Client(...)
        C:/Users/Komei_DiSanXian/go/pkg/mod/github.com/!komei!di!san!xian/floatbox@v0.0.0-20241008033815-74fe3739bb9e/web/http.go:22
github.com/FloatTech/AnimeAPI/ascii2d.ASCII2d({0xc00059eb60, 0xce})
        C:/Users/Komei_DiSanXian/go/pkg/mod/github.com/!float!tech/!anime!a!p!i@v1.7.0/ascii2d/ascii2d.go:28 +0xda5
github.com/FloatTech/ZeroBot-Plugin/plugin/saucenao.init.0.func2(0xc0027ddef0)
        E:/project/Go/RemiliaBot/plugin/saucenao/searcher.go:165 +0xc0e
github.com/wdvxdr1123/ZeroBot.match.func2.1()
        C:/Users/Komei_DiSanXian/go/pkg/mod/github.com/wdvxdr1123/!zero!bot@v1.7.5-0.20240501144516-eb574bbdad32/bot.go:256 +0x53
created by github.com/wdvxdr1123/ZeroBot.match.func2 in goroutine 27991
        C:/Users/Komei_DiSanXian/go/pkg/mod/github.com/wdvxdr1123/!zero!bot@v1.7.5-0.20240501144516-eb574bbdad32/bot.go:249 +0x99
```

--- a/go.mod
+++ b/go.mod
@@ -133,7 +133,7 @@ require (
 )
 
 replace (
-	github.com/FloatTech/floatbox => github.com/KomeiDiSanXian/floatbox v0.0.0-20241008033815-74fe3739bb9e
+	github.com/FloatTech/floatbox => github.com/KomeiDiSanXian/floatbox v0.0.0-20241009015604-05a7557b282d
 	github.com/FloatTech/zbputils => github.com/KomeiDiSanXian/zbputils v0.0.0-20240511052424-8b444ccfb843
 	github.com/remyoudompheng/bigfft => github.com/fumiama/bigfft v0.0.0-20211011143303-6e0bfa3c836b
 	modernc.org/sqlite => github.com/fumiama/sqlite3 v1.20.0-with-win386

